builder-support/debian/recursor/debian-buster/control

```diff
@@ -13,8 +13,8 @@ Build-Depends: debhelper-compat (= 10),
                libboost-thread-dev,
                libcap-dev [linux-any],
                libfstrm-dev,
-               libluajit-5.1-dev [amd64] | liblua5.1-0-dev,
-               liblua5.1-0-dev [!amd64],
+               libluajit-5.1-dev [amd64] | liblua5.3-dev,
```

Comment:
I suppose we want to replace this with the version in auth.

Reply:
Indeed, done.